pyscriptjs/src/main.ts

```diff
@@ -3,42 +3,73 @@ import './styles/pyscript_base.css';
 import { loadConfigFromElement } from './pyconfig';
 import type { AppConfig } from './pyconfig';
 import type { Runtime } from './runtime';
-import { PyScript } from './components/pyscript';
+import { PyScript, initHandlers, mountElements } from './components/pyscript';
 import { PyLoader } from './components/pyloader';
 import { PyodideRuntime } from './pyodide';
 import { getLogger } from './logger';
-import { globalLoader, runtimeLoaded, addInitializer } from './stores';
+import {
+    runtimeLoaded,
+    scriptsQueue,
+} from './stores';
 import { handleFetchError, showError, globalExport } from './utils'
+import { createCustomElements } from './components/elements';
+
 
 const logger = getLogger('pyscript/main');
 
-// XXX this should be killed eventually
-let runtimeSpec: Runtime;
-runtimeLoaded.subscribe(value => {
-    runtimeSpec = value;
+let scriptsQueue_: PyScript[];
+scriptsQueue.subscribe((value: PyScript[]) => {
+    scriptsQueue_ = value;
```

Comment:
I wonder if it's worth it to change these names? I think these they are so confusing... Do you have an idea for better naming?

Reply:
they are going to be killed soon :skull_and_crossbones: